package.json

```diff
@@ -104,10 +104,10 @@
     "gulp": "^4.0.2",
     "html-validate": "^7.3.0",
     "html-webpack-plugin": "^5.5.0",
-    "husky": "^6.0.0",
+    "husky": "^9.0.0",
     "jest": "^28.1.0",
     "jest-environment-jsdom": "^28.1.0",
-    "lint-staged": "^10.5.4",
+    "lint-staged": "^15.2.2",
```

Comment:
The linting kept failing for me for some reason. It thought I was committing an empty commit. So just updated this version as well 🤷🏻‍♂️